src/WorkflowCore/WorkflowCore.csproj

```diff
@@ -25,14 +24,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="1.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.0" />
```

Comment:
why do we need `Microsoft.AspNetCore.Http.Abstractions`?

Reply:
Because I have to use EF core 2.0 to fix some critical bugs that came from EF core 1.x. My project already updated to 2.0 and seems conflicted with `WorkflowCore.Persistence.EntityFramework 1.x` so my project cannot work normally.

This PR is not required. If one day it needs to update to 2.0, maybe this can make it easier :)

BTW, `Microsoft.AspNetCore.Http.Abstractions` is my mistake.